CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 2.1.0 (May 20, 2021)
+
+DEPRECATIONS:
+- **Deprecated Resource:** `metal_volume` ([#99](https://github.com/equinix/terraform-provider-metal/pull/99))
+- **Deprecated Resource:** `metal_volume_attachment` ([#99](https://github.com/equinix/terraform-provider-metal/pull/99))
+- **Deprecated Data Source:** `metal_volume` ([#99](https://github.com/equinix/terraform-provider-metal/pull/99))
+- removed unintended deprecation of `metal_device`.`facility` ([#82](https://github.com/equinix/terraform-provider-metal/pull/82))
+
+BUG FIXES:
+- removed deprecation of `metal_device`.`facility` ([#82](https://github.com/equinix/terraform-provider-metal/pull/82))
+- fixed plugin crash handling certain errprs ([#89](https://github.com/equinix/terraform-provider-metal/issues/89))
+
+FEATURES:
+- **New Resource:** `metal_connection` ([#76](https://github.com/equinix/terraform-provider-metal/pull/76))
+- **New Resource:** `metal_virtual_circuit`([#92](https://github.com/equinix/terraform-provider-metal/pull/92))
+- **New Data Source:** `metal_vlan` ([#67](https://github.com/equinix/terraform-provider-metal/pull/67))
+- **New Data Source:** `metal_reserved_ip_blcok`([#80](https://github.com/equinix/terraform-provider-metal/pull/80))
+- **New Data Source:** `metal_port`([#96](https://github.com/equinix/terraform-provider-metal/pull/96))
+- **New Data Source:** `metal_hardware_reservation`([#100](https://github.com/equinix/terraform-provider-metal/pull/100))
+- **New Guide:** Migrating From the Packet Provider ([#72](https://github.com/equinix/terraform-provider-metal/pull/72))
+- **New Guide:** Upgrades Devices from Facilities to Metros ([#103](https://github.com/equinix/terraform-provider-metal/pull/103))
+
+IMPROVEMENTS:
+
+- `metal_project` `md5_pass` field is now flagged sensitive ([#93](https://github.com/equinix/terraform-provider-metal/issues/93))
+- `metal_spot_market_request` `instance_parameters` are documented ([#104](https://github.com/equinix/terraform-provider-metal/issues/104))
+- added `metro` to `metal_connection` (`facility` now optional) ([#81](https://github.com/equinix/terraform-provider-metal/pull/81))
```

Comment:
```suggestion
- added `metro` to `metal_connection` datasource (`facility` now optional) ([#81](https://github.com/equinix/terraform-provider-metal/pull/81))
```

Reply:
@t0mk thanks - I've picked these suggestions up in https://github.com/equinix/terraform-provider-metal/pull/108